### PR TITLE
python3Packages.boost-histogram: init at 1.0.2

### PR DESCRIPTION
**Pull request by veprbl**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by veprbl** [comment]
@GrahamcOfBorg build python38Packages.boost-histogram python39Packages.boost-histogram

File: pkgs/development/python-modules/boost-histogram/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  propagatedBuildInputs = [ numpy ];
+
+  checkInputs = [ pytestCheckHook pytest-benchmark ];
+
+  meta = with lib; {
+    description = "Python bindings for the C++14 Boost::Histogram library";
+    homepage = "https://github.com/scikit-hep/boost-histogram";
+    license = lib.licenses.bsd3;
+    platforms = platforms.unix;
+    maintainers = with lib.maintainers; [ veprbl ];
```
Comment (jonringer) [inline pkgs/development/python-modules/boost-histogram/default.nix:24]: you have `with lib;` at the meta scope
```suggestion
    license = licenses.bsd3;
    platforms = platforms.unix;
    maintainers = with maintainers; [ veprbl ];
```

**Review by jonringer — COMMENTED** [review]
otherwise diff LGTM

**Comment by veprbl** [comment]
@GrahamcOfBorg build python38Packages.boost-histogram python39Packages.boost-histogram

**Comment by veprbl** [comment]
@GrahamcOfBorg build python38Packages.boost-histogram python39Packages.boost-histogram

**Pull request merged**